validphys2/src/validphys/core.py

```diff
@@ -793,10 +802,14 @@ def moment(self, order):
         return np.mean(np.power(self.error_members() - self.central_value(), order), axis=0)
 
     def errorbar68(self):
-        # Use nanpercentile here because we can have e.g. 0/0==nan normalization
-        # somewhere.
+        # Use nanpercentile here because we can have e.g. 0/0==nan normalization somewhere
         down = np.nanpercentile(self.error_members(), 15.87, axis=0)
         up = np.nanpercentile(self.error_members(), 84.13, axis=0)
+        # The central value is not necessarily the average of the error members
```

Comment:
```suggestion
        # The central value is not necessarily the median of the error members
```
It is instead _precisely_ the average of the error members.

For a gaussian distribution where we take enough samples (i.e. replicas)  the mean and median will be very close (distance decreasing as 1/sqrt(Nrep)) but if gaussianity breaks down and we have a small amount of samples the distance between mean and median can be large.